src/Display/Homepage.js

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect, useRef } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getPosts } from '../redux/slice/postSlice';
+import Movies from '../components/Details';
+import Header from '../components/Header';
+
+import '../index.css';
+
+const Shows = () => {
+  const posts = useSelector((state) => state.post.post);
+  const dispatch = useDispatch();
+
+  const shouldGetPosts = useRef(true);
+  useEffect(() => {
+    if (shouldGetPosts.current) {
+      shouldGetPosts.current = false;
+      dispatch(getPosts());
+    }
+    // eslint-disable-next-line
+    },[]);
+
+  return (
+    <div className="App">
+      <Header />
+
+      <div className="show-container">
+
+        {
+               posts.map((post) => (
+                 <Movies
+                   key={post.id}
+                   title={post.name}
+                   logo={post.logo_100px}
+                   id={post.id}
+                 />
+               ))
+         }
+      </div>
+    </div>
+  );
```

Comment:
Kindly ensure that your app matches the design in terms of the typography and layout (composition and space between elements). There seems to be some differences currently. 

- [ ] For instance the image is suppose to be at the left with the arrow in the top right corner and the text/title below.  
- [ ] The shape of the arrow is also different from that of the actual design as shown below. :point_down: 

### Your design 


![Screenshot from 2023-03-03 23-11-05](https://user-images.githubusercontent.com/65233860/222853039-26e57d37-b8fe-4598-a14c-a8e9cbb510ad.png)


### Actual design 


![Screenshot from 2023-03-03 23-11-39](https://user-images.githubusercontent.com/65233860/222853053-87806df6-2061-4704-b621-df0e116d207f.png)